ext/crypto/00_crypto.js

```diff
@@ -606,6 +657,76 @@
       }
     }
 
+    /**
+    * @param {AlgorithmIdentifier} algorithm
+    * @param {CryptoKey} baseKey
+    * @param {number} length
+    * @returns {Promise<ArrayBuffer>}
+    */
+    async deriveBits(algorithm, baseKey, length) {
+      webidl.assertBranded(this, SubtleCrypto);
+      const prefix = "Failed to execute 'deriveBits' on 'SubtleCrypto'";
+      webidl.requiredArguments(arguments.length, 3, { prefix });
+      algorithm = webidl.converters.AlgorithmIdentifier(algorithm, {
+        prefix,
+        context: "Argument 1",
+      });
+      baseKey = webidl.converters.CryptoKey(baseKey, {
+        prefix,
+        context: "Argument 2",
+      });
+      length = webidl.converters["unsigned long"](length, {
+        prefix,
+        context: "Argument 3",
+      });
+
+      const normalizedAlgorithm = normalizeAlgorithm(algorithm, "deriveBits");
+      switch (normalizedAlgorithm.name) {
+        case "PBKDF2": {
+          // 1.
+          if (length == null || length == 0 || length % 8 !== 0) {
```

Comment:
Where does it say that length may not be 0?

Reply:
Not in the spec but WPT has a test for it https://github.com/web-platform-tests/wpt/blob/561fc4c5463673e5291894705c7a6aeb88e78d99/WebCryptoAPI/derive_bits_keys/pbkdf2.js#L118

Reply:
Can you open a spec issue?

Reply:
Sure - https://github.com/w3c/webcrypto/issues/274